From hjk@linutronix.de  Mon Jun 14 15:26:29 2010
From: "Hans J. Koch" <hjk@linutronix.de>
Date: Thu, 10 Jun 2010 01:15:49 +0200
Subject: uio: Remove IRQF_DISABLED from uio_sercos3.c
Cc: John Ogness <john.ogness@linutronix.de>, Greg KH <gregkh@suse.de>
Message-ID: <20100609231549.GD2555@local>
Content-Disposition: inline


Remove IRQF_DISABLED since it is deprecated and a no-op in the
current kernel.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
Acked-by: John Ogness <john.ogness@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uio/uio_sercos3.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/uio/uio_sercos3.c
+++ b/drivers/uio/uio_sercos3.c
@@ -154,7 +154,7 @@ static int __devinit sercos3_pci_probe(s
 	info->name = "Sercos_III_PCI";
 	info->version = "0.0.1";
 	info->irq = dev->irq;
-	info->irq_flags = IRQF_DISABLED | IRQF_SHARED;
+	info->irq_flags = IRQF_SHARED;
 	info->handler = sercos3_handler;
 	info->irqcontrol = sercos3_irqcontrol;
 
